components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -61,74 +71,83 @@ export default class SchoolCard extends Component {
+            <span className={styles.cardBlockTitle}>Availability</span>
+            <div className={styles.badgeGroup}>
+              <Badge
+                label={<LabelWithScreenReader label="Online" isActive={props.hasOnline} />}
+                icon={<OnlineIcon />}
+                className={badgeClassNames(props.hasOnline)}
+              />
+              <Badge
+                label={<LabelWithScreenReader label="Campus" isActive={!props.hasOnlyOnline} />}
+                icon={<CampusIcon />}
+                className={badgeClassNames(!props.hasOnlyOnline)}
+              />
+              <Badge
+                label={<LabelWithScreenReader label="Housing" isActive={props.hasHousing} />}
+                icon={<HousingIcon />}
+                className={badgeClassNames(props.hasHousing)}
+              />
+              <Badge
+                label={
+                  <LabelWithScreenReader label="Equipment" isActive={props.hasHardwareIncluded} />
+                }
+                icon={<EquipmentIcon />}
+                className={badgeClassNames(props.hasHardwareIncluded)}
+              />
+            </div>
           </div>
-        </div>
 
-        <div className={styles.cardBlock}>
-          <span className={styles.cardBlockTitle}>Accepts GI Bill</span>
-          {hasGiBill ? 'Yes' : 'No'}
-        </div>
+          <div className={styles.cardBlock}>
+            <span className={styles.cardBlockTitle}>Accepts GI Bill</span>
+            {hasGiBill ? 'Yes' : 'No'}
+          </div>
 
-        <div className={styles.cardBlock}>
-          <span className={styles.cardBlockTitle}>Campus Locations</span>
-          {getSchoolLocationText(props.hasOnlyOnline, props.locations)}
-          {props.locations.length > 1 && (
-            <>
-              {' ('}
-              <Button
-                analyticsObject={{
-                  action: 'Button Selected',
-                  category: 'Interactions',
-                  label: `${props.name} | Locations`,
-                }}
-                onClick={this.toggleModalClick}
-                className={styles.modalToggler}
-              >
-                view all
-              </Button>
-              {')'}
-            </>
-          )}
-        </div>
+          <div className={styles.cardBlock}>
+            <span className={styles.cardBlockTitle}>Campus Locations</span>
+            {getSchoolLocationText(props.hasOnlyOnline, props.locations)}
+            {props.locations.length > 1 && (
+              <>
+                {' ('}
+                <Button
+                  analyticsObject={{
+                    action: 'Button Selected',
+                    category: 'Interactions',
+                    label: `${props.name} | Locations`,
+                  }}
+                  onClick={this.toggleModalClick}
+                  className={styles.modalToggler}
+                >
+                  view all
+                </Button>
+                {')'}
```

Comment:
you can just throw the parens into the Button child. It'll look a bit cleaner cuz you can drop the fragments and string templates, and it's okay if that aspect doesn't match the mock flawlessly (underlines under the parens)

Reply:
How do you feel about just making it look like a button? I wasn't a fan of parentheses, and with it having an underline: the parentheses would cut off the underline and leave the tips of line's start/end.
![Button example](https://user-images.githubusercontent.com/2691129/56832353-be801380-6839-11e9-9484-66bd54daad4d.png)

Reply:
Sure that looks fine to me :)

Just do your best with handling hover/focus states. I trust ya!